pocs/utils/rs232.py

```diff
@@ -148,13 +158,27 @@ def get_reading(self):
 
     def clear_buffer(self):
         """ Clear Response Buffer """
-        count = 0
-        while self.ser.inWaiting() > 0:
-            count += 1
-            self.ser.read(1)
-
+        # Not worrying here about bytes arriving between reading in_waiting
+        # and calling reset_input_buffer().
+        # Note that Wilfred reports that the Arduino input can seriously lag behind
```

Comment:
The lag of 10 seconds would probably apply more to the output buffer. Once it is lagging the only way to catch-up would be to clear output buffer.

Reply:
output meaning writing from POCS to the Arduino? That sounds like a flush call needs to be made on the device to ensure it is making it out of buffers and into the OS.

Reply:
Output meaning from the Arduino to POCS. Unless we connect to and start reading from the arduino immediately after it is reset the sensor readings lag behind.

Reply:
Let's try to be specific when we use the words input and output (I wasn't sufficiently in the comment). So, "input from Arduino" rather than "Arduino input", which was ambiguous; or "POCS output to the mount".

Reply:
I've updated the comment accordingly.